README.md

```diff
@@ -1,6 +1,81 @@
+
+[![Git Releases](https://img.shields.io/github/v/release/mojaloop/helm?label=helm%20version)](https://github.com/mojaloop/helm/releases)
+[![License](https://img.shields.io/badge/Licence-Apache%202.0-orange.svg)](./LICENSE.md)
+
+## Documentation
+- [Official Mojaloop Documentation](https://docs.mojaloop.io/documentation)
+- [Mojaloop Specification](https://github.com/mojaloop/mojaloop-specification)
+- [Developer Onboarding Guide](./onboarding.md)
+- [Deploy Mojaloop On Kubernetes](https://docs.mojaloop.io/documentation/deployment-guide/)
+- [Mojaloop Business Documentation](https://docs.mojaloop.io/mojaloop-business-docs/)
+
+## Contact
+
+[Slack](https://mojaloop-slack.herokuapp.com/) is the best way to keep in touch with the Mojaloop Community. Sign up for your Mojaloop Slack account [here](https://mojaloop-slack.herokuapp.com/).
+
+__Channels:__
+- `#announcements` - Announcements for new Releases and QA Status
+- `#design-authority`- Questions + Discussion around Mojaloop Design
+- `#general` - General discussion about Mojaloop
+- `#help-mojaloop` - Ask for help with installing or running Mojaloop
+- `#ml-oss-bug-triage` - Discussion and triage for new bugs and issues
+
+## Contributing
+
+Refer to the [Contributors guide](https://docs.mojaloop.io/documentation/contributors-guide/) to get started making contributions.
+
+You may also want to read:
+- [Mojaloop Github Project](https://github.com/mojaloop/project)
+  > _note: You must install the [ZenHub extension](https://www.zenhub.com/extension) in order to see the Project Kanban Board_
+- [Developer Onboarding Guide](./onboarding.md)
+
+If you have any trouble getting started, or want to know where to start, reach out to us on the `#general` channel in [Slack](https://mojaloop-slack.herokuapp.com/)!
+
+### Mojaloop Repositories
+
+Refer to the [Mojaloop Repository Overview](https://docs.mojaloop.io/documentation/repositories/) for a detailed breakdown of each Mojaloop Repository in development.
+
+As a quick reference, some of the most important repositories for contributors are:
+
+- [central-ledger](https://github.com/mojaloop/central-ledger) - Core Service in charge of managing transfers between DFSPs
+- [ml-api-adapter](https://github.com/mojaloop/ml-api-adapter) - Translation layer to convert to/from Mojaloop API to an internal format that is used in Central Services Stack.
+- [account-lookup-service](https://github.com/mojaloop/account-lookup-service) - A service for managing party accounts across a Mojaloop environment
+- [helm](https://github.com/mojaloop/helm) - The Helm chart development for deploying Mojaloop on Kubernetes
+- [postman](https://github.com/mojaloop/postman) - A set of Postman scripts for testing and managing a Mojaloop environment
+- [documentation](https://github.com/mojaloop/documentation) - The mojaloop documentation, published [here](https://docs.mojaloop.io/documentation)
```

Comment:
Can you please add https://github.com/mojaloop/ml-testing-toolkit and https://github.com/mojaloop/project repositories as well :-)